src/main/game_globals.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * TheXTech - A platform game engine ported from old source code for VB6
+ *
+ * Copyright (c) 2009-2011 Andrew Spinks, original VB6 code
+ * Copyright (c) 2020-2021 Vitaly Novichkov <admin@wohlnet.ru>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef GAME_GLOBALS_H
+#define GAME_GLOBALS_H
+
+#include "../screen_fader.h"
+
+extern ScreenFader g_levelScreenFader;
```

Comment:
Why here and not in `globals.cpp`?

Reply:
Because I desired to start the gradual organizing of the thing and making "globals" thing stop to grow, probably later going to split it down into multiple parts (mainly, public structures such as player, block, NPC, etc., etc.)